### PR TITLE
Update README.markdown

### PR DESCRIPTION
**Pull request by KevinBongart**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 It's just a Kanban board made with Ruby on Rails, but it's pretty. More information in [this blog post](http://wedontneedroads.net/post/15571399433/kaban-web-app).
 
-**Warning: this app hasn't been maintained since [Trello](https://trello.com) came out. Fork and improve for fun but, other than that, user Trello.**
+**Warning: this app hasn't been maintained since [Trello](https://trello.com) came out. Fork and improve for fun but, other than that, use Trello. ❤️**
```
Comment (midu) [inline README.markdown]: does it work on chrome?

Comment (KevinBongart) [inline README.markdown]: no but i made sure it works on opera

Comment (midu) [inline README.markdown]: feels like writing that line makes it a lie

Comment (midu) [inline README.markdown]: I only use this app for its readme

**Pull request merged**